docs/website/docs/reference/installation.md

```diff
@@ -87,7 +89,7 @@ source ./env/bin/activate
   </TabItem>
   <TabItem value="windows">
 
-Create a new virtual environment by making a `./env` directory to hold it.
+Create a new virtual environment in your working folder. This will create an `./env` directory where your virtual environment will be stored
```

Comment:
```suggestion
Create a new virtual environment in your working folder. This will create an `./env` directory where your virtual environment will be stored:
```